pymc/logprob/mixture.py

```diff
@@ -289,9 +289,10 @@ def find_measurable_index_mixture(fgraph, node):
         # We don't support (non-scalar) integer array indexing as it can pick repeated values,
         # but the Mixture logprob assumes all mixture values are independent
         if any(
-            indices.dtype.startswith("int") and sum(1 - b for b in indices.type.broadcastable) > 0
+            hasattr(indices, "dtype")
```

Comment:
You can check `isinstance(indices, TensorVariable)` and then it's guaranteed to have a dtype and a broadcastable